aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -278,6 +302,12 @@ Using the stdout log is only recommended for troubleshooting app startup issues.
 
 A timestamp and file extension are added automatically when the log file is created. The log file name is composed by appending the timestamp, process ID, and file extension (*.log*) to the last segment of the `stdoutLogFile` path (typically *stdout*) delimited by underscores. If the `stdoutLogFile` path ends with *stdout*, a log for an app with a PID of 1934 created on 2/5/2018 at 19:42:32 has the file name *stdout_20180205194132_1934.log*.
 
+::: moniker range=">= aspnetcore-2.2"
+
+If `stdoutLogEnabled` is false, errors that occur on application startup will be captured and emitted to the event log up to 30KB. After startup, all additional logs are discarded.
```

Comment:
* "application" :point_right: "app"
* "will be captured" :point_right: "are captured" (present tense)
* Space between "30" and "KB" ... "30 KB"

Cut-'n-paste ...

> If \`stdoutLogEnabled\` is false, errors that occur on app startup are captured and emitted to the event log up to 30 KB. After startup, all additional logs are discarded.